docs/_includes/sections.adoc

```diff
@@ -32,7 +32,7 @@ include::ex-section.adoc[tags=base]
 include::ex-section.adoc[tags=b-base]
 ====
 
-TIP: In addition to the equals marker used for defining single-line section titles, Asciidoctor recognizes the hash symbol (+#+) from Markdown.
+TIP: In addition to the equal sign marker used for defining single-line section titles, Asciidoctor recognizes the hash symbol (+#+) from Markdown.
```

Comment:
I'm pretty sure the terminology for this should just be "equals sign" (that's how it's named in Unicode).